torch/jit/_shape_functions.py

```diff
@@ -837,6 +837,22 @@ def flatten(input: List[int], start_dim: int, end_dim: int):
         shape.append(input[i])
     return shape
 
+def _reduce_along_dim(self: List[int], dim: int, keepdim: bool):
+    dim = maybe_wrap_dim(dim, len(self))
+    out: List[int] = []
+    for i, self_dim in enumerate(self):
+        if i == dim:
+            if keepdim:
```

Comment:
Can you add `assert_jit_shape_analysis` to `
```
ReductionOpInfo(
        'argmax',
        supports_multiple_dims,
        ...
)
```
in 
https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_methods_invocations.py

wont let me link bc the file is too large 😢